cloud/aws/services/ec2/routetables.go

```diff
@@ -142,12 +144,19 @@ func (s *Service) describeVpcRouteTables(clusterName string, vpcID string) ([]*e
 	return out.RouteTables, nil
 }
 
-func (s *Service) createRouteTableWithRoutes(clusterName string, vpc *v1alpha1.VPC, routes []*ec2.Route) (*v1alpha1.RouteTable, error) {
+func (s *Service) createRouteTableWithRoutes(clusterName string, vpc *v1alpha1.VPC, routes []*ec2.Route, isPublic bool) (*v1alpha1.RouteTable, error) {
```

Comment:
This and `createSubnet` look very similar. It's ok to copy pasta them for now, but one is getting isPublic passed in and the other is calculating it, might be something to clean up later

Reply:
Added a comment for later cleanup